homeassistant/components/utility_meter/sensor.py

```diff
@@ -418,3 +468,28 @@ def extra_state_attributes(self):
     def icon(self):
         """Return the icon to use in the frontend, if any."""
         return ICON
+
+    @property
+    def last_reset(self):
+        """Return the last time the sensor was reset."""
+        return self._last_reset
```

Comment:
Is this change related to this PR or is it a separate thing?
There's still a string version of last reset being set in the sensor's extra attributes above.

Reply:
Can't recall anymore...